terraform/a2ha-terraform/modules/efs/inputs.tf

```diff
@@ -0,0 +1,67 @@
+variable "automate_private_ips" {
+  default = []
+}
+
+variable "aws_region" {
+  default     = "us-west-2"
```

Comment:
this can be override ?

Reply:
yes this will be overridden on basis of config.toml input